src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/magical/MagicalZombiePills.java

```diff
@@ -47,7 +51,8 @@ public EntityInteractHandler getItemHandler() {
         return (e, item, offhand) -> {
             Entity entity = e.getRightClicked();
 
-            if (entity.getType() == EntityType.ZOMBIE_VILLAGER) {
+            if (SlimefunPlugin.getMinecraftVersion().isAtLeast(MinecraftVersion.MINECRAFT_1_16) && entity instanceof PigZombie
+                    || entity instanceof ZombieVillager) {
```

Comment:
Not a fan of this line here, it's really messy and redundant with the two if statements below.
I'll suggest an alternative below